src/Main.java

```diff
@@ -26,6 +28,7 @@ public static void main(String[] args) throws FileNotFoundException {
     String Udata;
     String Pdata;
     int num;
+    int num1;
```

Comment:
Please have some meaningful variable, variable name like this doesn't make sense to anyone